src/openrct2-ui/windows/RideConstruction.cpp

```diff
@@ -1458,6 +1458,7 @@ static Widget _rideConstructionWidgets[] = {
                     break;
                 case TrackElemType::BlockBrakes:
                 case TrackElemType::DiagBlockBrakes:
+                case TrackElemType::SlopedBrakes:
```

Comment:
This is only there to gently encourage people to build block brakes with default speed for TD6 compatibility. It's not necessary for brakes.

Reply:
Thanks! I saw "brakes" and got excited.